feature-account-impl/src/main/java/jp/co/soramitsu/feature_account_impl/presentation/about/AboutViewModel.kt

```diff
@@ -22,12 +22,33 @@ class AboutViewModel(
     private val _websiteLiveData = MutableLiveData<String>()
     val websiteLiveData: LiveData<String> = _websiteLiveData
 
+    private val _twitterLiveData = MutableLiveData<String>()
```

Comment:
Why instead of coupling vars not give a ViewModel an interface which allows get-access with LiveData type, and get rid of wall of redundant vars?

Reply:
Maybe we can move these social media links from BuildConfig to another location and on the presentation layer show it as RecyclerView? This will save us a lot of variables here and adding more links would be easier.